frontend/src/entities/configuration/project-creation-configuration.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License").
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import {
+    Container,
+    Header,
+    Toggle,
+    Grid
+} from '@cloudscape-design/components';
+import React from 'react';
+
+export type ProjectCreationConfigurationProps = {
+    setFields: (any) => void;
+    isAdminOnly: boolean;
+};
+
+export function ProjectCreationConfiguration (props: ProjectCreationConfigurationProps) {
+    return (
+        <>
+            <Container
+                header={
+                    <Header variant='h2'
+                        description='All users can create new projects in MLSpace. Restrict project creation to users with Admin permissions by activating “Admin Only.”'
+                    >
+                        Project Creation
+                    </Header>
+                }>
+                <Grid gridDefinition={[{ colspan: 2 }]}>
+                    <div style={{display: 'grid', textAlign: 'center'}}>
+                        <div style={{display: 'grid', justifyContent: 'center'}}>
+                            <Toggle
```

Comment:
Why not use the label and description from the toggle? Just for visual purposes or is there another reason?
```
<Toggle
      onChange={({ detail }) =>
        rops.setFields({'configuration.ProjectCreation.isAdminOnly': detail.checked});
      }
      checked={props.isAdminOnly}
      description={
          Restrict project creation to users with Admin
          permissions
      }
    >
      Admin Only
    </Toggle>
```
![Screenshot 2024-06-11 at 11 40 22 AM](https://github.com/awslabs/mlspace/assets/28429388/c2b06cfd-41dc-4fa5-ab46-146ecc75b8b6)

Reply:
My task said to have the items it is toggling off below the toggle, this was the way to accomplish this. I can look into using the box, space-between components you called out. I didn't try with those yet. Not sure it will accomplish what we are looking for or not.